By Omair Majid. Obtained from Alpine Linux.

Notably, Dotnet-9 already has these.

	-mi
--- src/runtime/src/coreclr/vm/comreflectioncache.hpp	2024-05-28 12:53:38.000000000 -0400
+++ src/runtime/src/coreclr/vm/comreflectioncache.hpp	2025-04-28 21:59:11.041285000 -0400
@@ -27,4 +27,5 @@
     void Init();
 
+#ifndef DACCESS_COMPILE
     BOOL GetFromCache(Element *pElement, CacheType& rv)
     {
@@ -103,4 +104,5 @@
         this->LeaveWrite();
     }
+#endif // !ACCESS_COMPILE
 
 private:
@@ -142,4 +144,5 @@
     }
 
+#ifndef DACCESS_COMPILE
     void AdjustStamp(BOOL hasWriterLock)
     {
@@ -171,4 +174,5 @@
             this->LeaveWrite();
     }
+#endif // !ACCESS_COMPILE
 
     void UpdateHashTable(SIZE_T hash, int slot)
